clients/cloud/python/README.md

```diff
@@ -0,0 +1,67 @@
+# Overview
+
+Produce messages to and consume messages from [Confluent Cloud](https://www.confluent.io/confluent-cloud/) using [Confluent Python Client for Apache Kafka](https://github.com/confluentinc/confluent-kafka-python).
+
+
+# Prerequisites
+
+* [Confluent's Python Client for Apache Kafka](https://github.com/confluentinc/confluent-kafka-python) installed on your machine
+* Access to a [Confluent Cloud](https://www.confluent.io/confluent-cloud/) cluster
+* Local file with configuration parameters to connect to your Confluent Cloud instance ([how do I find those?](https://docs.confluent.io/current/cloud/using/config-client.html#librdkafka-based-c-clients)). Format the file as follows:
+
+
+```bash
+$ cat ~/.ccloud/librdkafka.config
+bootstrap.servers=<broker-1,broker-2,broker-3>
+sasl.username=<api-key-id>
+sasl.password=<secret-access-key>
+```
+
+# Example 1: Hello World!
+
+In this example, the producer writes Kafka data to a topic in Confluent Cloud. 
+Each record has a key representing a username (e.g. `alice`) and a value of a count, formatted as json (e.g. `{"count": 0}`).
+The consumer reads the same topic from Confluent Cloud and keeps a rolling sum of the counts as it processes each record.
+
+1. Run the producer, passing in arguments for (a) the local file with configuration parameters to connect to your Confluent Cloud instance and (b) the topic name:
+
+```bash
+$ ./producer.py -f ~/.ccloud/librdkafka.config -t test1
+Preparing to produce record: alice 	 {"count": 0}
+Preparing to produce record: alice 	 {"count": 1}
+Preparing to produce record: alice 	 {"count": 2}
+Preparing to produce record: alice 	 {"count": 3}
+Preparing to produce record: alice 	 {"count": 4}
+Preparing to produce record: alice 	 {"count": 5}
+Preparing to produce record: alice 	 {"count": 6}
+Preparing to produce record: alice 	 {"count": 7}
+Preparing to produce record: alice 	 {"count": 8}
+Preparing to produce record: alice 	 {"count": 9}
+Successfully produced record to topic test1 partition [0] @ offset None
```

Comment:
This printout doesn't look right, there should a proper offset printed.
What py client and librdkafka versions are you on?